src/org/parosproxy/paros/extension/option/OptionsParamView.java

```diff
@@ -453,24 +472,24 @@ public void setLargeResponseSize(int largeResponseSize) {
 		getConfig().setProperty(LARGE_RESPONSE_SIZE, largeResponseSize);
 	}
 
-	public int getFontSize() {

```

Comment:
We should keep the old method (and deprecate it) to not break binary compatibility. Same for other public methods.

Reply:
Yes, the four methods.